tools/nite/main.cpp

```diff
@@ -0,0 +1,513 @@
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#define _NBL_STATIC_LIB_
```

Comment:
does this define even make sense anymore

Reply:
ah oldie shit yes, still there XD

Reply:
copy paste magic